__tests__/test.js

```diff
@@ -40,8 +39,35 @@ describe('/api/topics', () => {
 
 describe('/api', () => {
     test('GET request should respond with 200 and an object containing all endpoints with a description', () => {
-        return request(app).get('/api').then(({body}) => {
-            expect(body.endpoints).toEqual(endpoints)
-            })
+    return request(app).get('/api').expect(200).then(({body}) => {
+        expect(body.endpoints).toEqual(endpoints)
+        })
+    })
+})
+
+describe('/api/articles/:article_id', () => {
+    test('GET request should respond with 200 and an object', () => {
+        return request(app).get('/api/articles/4').expect(200).then(({body}) => {
+            const {article} = body
+            expect(typeof article).toBe('object')
         })
     })
+    test(`GET request should respond with an object containing the folowing properties:author, title, article_id, body, topic, created_at, votes, article_img_url`, () => {
+        return request(app).get('/api/articles/4').then(({body}) => {
+            const {article} = body
+            expect(article).toHaveProperty('author')
+            expect(article).toHaveProperty('title')
+            expect(article).toHaveProperty('body')
+            expect(article).toHaveProperty('topic')
+            expect(article).toHaveProperty('created_at')
+            expect(article).toHaveProperty('votes')
+            expect(article).toHaveProperty('article_img_url')
+            expect(article.article_id).toBe(4)
+        })
```

Comment:
It's very good to see the `article_id` you are expecting to hold a specific value of `4`, the values for the other properties are also written in the test data and could also be used in this test

A jest matcher which can be really useful in a situation like this is `toMatchObject` docs [here](https://jestjs.io/docs/expect#tomatchobjectobject)